### PR TITLE
Remove deprecated $ symbol in punctuation.rst

### PR DESCRIPTION
**Pull request by kshyatt**

### CODE REVIEW

File: doc/stdlib/punctuation.rst [hunk 1]
```diff
@@ -13,7 +13,7 @@ symbol      meaning
 ``#``       begin single line comment
 ``#=``      begin multi-line comment (these are nestable)
 ``=#``      end multi-line comment
-``$``       bitwise xor operator, string and expression interpolation
+``⊻``       bitwise xor operator, string and expression interpolation
```
Comment (carlobaldassi) [inline doc/stdlib/punctuation.rst]: The "string and expression interpolation" does not apply to `⊻`, `$` should be kept for that.

Comment (kshyatt) [inline doc/stdlib/punctuation.rst]: So split into two lines, then?

Comment (carlobaldassi) [inline doc/stdlib/punctuation.rst]: Yes, I think `$` should stay where it is now (minus the xor description), and the new `⊻` should be added below, probably after `&` and `|`.

Comment (kshyatt) [inline doc/stdlib/punctuation.rst]: Sounds good!

**Comment by kshyatt** [comment]
@carlobaldassi good to go now?

**Review by carlobaldassi — APPROVED** [review]

**Pull request merged**